Figgle.Generator/FiggleSourceGenerator.cs

```diff
@@ -104,6 +106,9 @@ public void Execute(GeneratorExecutionContext context)
     {
         context.AddSource("GenerateFiggleTextAttribute.cs", AttributeSource);
 
+        var additionalFiles = context.AdditionalFiles.ToImmutableDictionary(
```

Comment:
There could be many `AdditionalFiles`, with only some of them containing Figgle font files. How about filtering to only include `.flf` files, to avoid accidentally picking up unexpected files that happen to have the same name but different extensions.

Reply:
Good point!  I've updated the string comparison to check the path's file name against `$"{fontName}.flf"`.